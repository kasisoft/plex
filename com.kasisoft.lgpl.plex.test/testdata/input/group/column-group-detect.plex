<?xml version="1.0" encoding="UTF-8"?>
<plex>

  <general>
    <interface api="transform"  id="cleanup"          classname="com.kasisoft.lgpl.plex.impl.CleanupTransform"      />
    <interface api="column"     id="simpleresolver"   classname="com.kasisoft.lgpl.plex.impl.SimpleColumnResolver"  />
  </general>

  <sheet name="column-group-detect" firstrow="0">
    <columngroup count="2">
      <columndetect refid="simpleresolver">
      </columndetect>
      <member title="name"       offset="0"/>
      <member title="callname"   offset="1"/>
      <member title="age"        offset="2"/>
    </columngroup>
  </sheet>
  
</plex>