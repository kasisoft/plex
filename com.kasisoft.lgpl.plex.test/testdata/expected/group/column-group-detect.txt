<workbook>
  <sheet name="column-group-detect" columns="6" rows="5">
    <column name="name[0]" type="java.lang.String"/>
    <column name="callname[0]" type="java.lang.String"/>
    <column name="age[0]" type="java.lang.Object"/>
    <column name="name[1]" type="java.lang.String"/>
    <column name="callname[1]" type="java.lang.String"/>
    <column name="age[1]" type="java.lang.Object"/>
    <row row="0">
      <name[0]>Name</name[0]>
      <callname[0]>Callname</callname[0]>
      <age[0]>Age</age[0]>
      <name[1]>Name</name[1]>
      <callname[1]>Callname</callname[1]>
      <age[1]>Age</age[1]>
    </row>
    <row row="1">
      <name[0]>Astair</name[0]>
      <callname[0]>Fred</callname[0]>
      <age[0]>28.0</age[0]>
      <name[1]>Mercury</name[1]>
      <callname[1]>Freddy</callname[1]>
      <age[1]>47.0</age[1]>
    </row>
    <row row="2">
      <name[0]>May</name[0]>
      <callname[0]>Brian</callname[0]>
      <age[0]>67.0</age[0]>
      <name[1]>Michael</name[1]>
      <callname[1]>George</callname[1]>
      <age[1]>53.0</age[1]>
    </row>
    <row row="3">
      <name[0]>Minogue</name[0]>
      <callname[0]>Kylie</callname[0]>
      <age[0]>43.0</age[0]>
      <name[1]>Vega</name[1]>
      <callname[1]>Suzanne</callname[1]>
      <age[1]>51.0</age[1]>
    </row>
    <row row="4">
      <name[0]>Bradfield</name[0]>
      <callname[0]>James Dean</callname[0]>
      <age[0]>54.0</age[0]>
      <name[1]/>
      <callname[1]/>
      <age[1]/>
    </row>
  </sheet>
</workbook>
